man/calculateFragments-methods.Rd

```diff
@@ -61,9 +61,11 @@
       Currently \code{tolerance}, \code{method} and \code{relative} are
       supported.
       You could change the \code{tolerance} (default \code{25e-6}) and
-      decide whether this tolerance should be applied relative
+      decide whether this tolerance should be applied relative to the target m/z
       (default \code{relative = TRUE}) or absolute (\code{relative = FALSE})
-      to match the theoretical fragment MZ with the MZ of the spectrum.
+      to match the theoretical fragment MZ with the MZ of the spectrum. When
+      (\code{relative = TRUE}) the mass tolerance window is set to target mz
+      +/- (target mz * (\code{tolerance})) and target mz +/- (\code{tolerance}) otherwise.
```

Comment:
Could you use `\code{target mz +/- (target mz * tolerance)}` and `\code{target mz +/- tolerance}`?